blackduck/Client.py

```diff
@@ -29,24 +75,19 @@ class Client:
 
     def __init__(
         self,
-        *args,
```

Comment:
*args and **kwargs not used so I removed them. Also all arguments to Client should be explicitly named so that people can find them more easily if they look at the source.